vivo-project/VIVO
# [i18n-3762] Moving UI labels translations into vivo

**[VIVO GitHub issue](https://github.com/vivo-project/VIVO/issues)**: [3762](https://github.com/vivo-project/VIVO/issues/3762)

[Companion PR](https://github.com/vivo-project/Vitro/pull/350)

# What does this pull request do?
Moving UI labels translations from VIVO-languages into VIVO (+ adoption of a pom file)

# What's new?
webapp directories from VIVO-languages repository are not needed any more. Together with [PR](https://github.com/vivo-project/VIVO/pull/3797) for doing the same for home directories from Vitro-languages should enable archiving the VIVO-languages repositories.

# How should this be tested?

- Checkout https://github.com/vivo-project/VIVO-languages/tree/i18n-redesign as VIVO-languages project
- Remove VIVO home and install the new, fresh instance of VIVO
- Validate it is working as expected
- Try switching between all 8 languages (set in the runtime.properties languages.selectableLocales = en_US, de_DE, sr_Latn_RS, ru_RU, fr_CA, en_CA, es, pt_BR)
- 

# Interested parties
Tag (@ mention) interested parties or, if unsure, @VIVO-project/vivo-committers
